Apsitvarkom.Models/Mapping/PollutedLocationProfile.cs

```diff
@@ -15,9 +15,9 @@ public PollutedLocationProfile()
         // PollutedLocationDTO to PollutedLocation
         CreateMap<PollutedLocationDTO, PollutedLocation>()
             .ForMember(dest => dest.Id, opt => opt
-                .MapFrom(src => Guid.Parse(src.Id)))
+                .MapFrom(src => Guid.Parse(src.Id ?? string.Empty)))
             .ForMember(dest => dest.Spotted, opt => opt
-                .MapFrom(src => DateTime.Parse(src.Spotted, CultureInfo.InvariantCulture)));
+                .MapFrom(src => DateTime.Parse(src.Spotted ?? string.Empty, CultureInfo.InvariantCulture)));
```

Comment:
Same for `src.Spotted`

Reply:
Possible null reference warning. Even though we will put it through validation, I'll keep it here for now.